app/vmui/packages/vmui/src/components/LineChart/LineChart.tsx

```diff
@@ -1,196 +1,127 @@
+  };
 
-          // define closest series in chart
-          const hoveringOverValue = yScale.invert(y);
-          const closestPoint = series.map(s => s.values[key]?.value).reduce((acc, nextValue, index) => {
-            const delta = Math.abs(hoveringOverValue - nextValue);
-            if (delta < acc.delta) {
-              acc = {delta, index};
-            }
-            return acc;
-          }, {delta: Infinity, index: 0});
+  const onPanComplete = ({chart}: {chart: Chart}) => {
+    const {min, max} = getRangeTimeScale(chart);
+    if (!min || !max) return;
+    const {start,  end} = getTimeperiodForDuration(duration, new Date(max));
+    const period: TimePeriod = {
+      from: dateFromSeconds(start),
+      to: dateFromSeconds(end)
+    };
+    dispatch({type: "SET_PERIOD", payload: period});
+  };
 
-          const date = dateFromSeconds(series[0].values[key].key);
-          // popover orientation should be defined based on the scale domain middle, not data, since
-          // data may not be present for the whole range
-          const leftPart = date.valueOf() < (xScale.domain()[1].valueOf() + xScale.domain()[0].valueOf()) / 2;
-          setTooltipState({
-            date,
-            xCoord: xScale(date),
-            index: key,
-            activeSeries: closestPoint.index,
-            leftPart
-          });
-          setShowTooltip(true);
+  const options: ChartOptions = {
+    animation: {
+      delay: 0,
+      duration: 250,
+      easing: "linear",
+    },
+    scales: {
+      x: {
+        type: "time",
+        time: {
+          tooltipFormat: "yyyy-MM-dd HH:mm:ss.SSS",
+          displayFormats: {
+            millisecond: "HH:mm:ss.SSS",
+            second: "HH:mm:ss",
+            minute: "HH:mm",
+            hour: "HH:mm"
+          },
+        },
+        ticks: {
+          source: "auto",
+          autoSkip: true,
         }
-      } else {
-        setShowTooltip(false);
-        setTooltipState(undefined);
       }
     },
-    [xScale, yScale, series]
-  );
-
-  const tooltipData: ChartTooltipData | undefined = useMemo(() => {
-    if (tooltipState?.activeSeries) {
-      return {
-        value: series[tooltipState.activeSeries].values[tooltipState.index].value,
-        metrics: categories.map(c => ({ key: c.key, value: series[tooltipState.activeSeries].metric[c.key]}))
-      };
-    } else {
-      return undefined;
-    }
-  }, [tooltipState, series]);
-
-  const tooltipAnchor = useRef<SVGGElement>(null);
-
-  const seriesDates = useMemo(() => {
-    if (series && series[0]) {
-      return series[0].values.map(v => v.key).map(dateFromSeconds);
-    } else {
-      return [];
+    plugins: {
+      legend: {
+        position: "bottom",
+        align: "start",
+        labels: {
+          padding: 20,
+          color: "#000"
+        }
+      },
+      zoom: {
+        pan: {
+          enabled: true,
+          mode: "x",
+          onPanComplete:  debounce(onPanComplete, 1500)
```

Comment:
these 1.5 seconds add to subjective slowness of a chart

Reply:
fix to 500ms